tern/classes/docker_image.py

```diff
@@ -165,3 +165,10 @@ def load_image(self):
                 e.returncode, cmd=e.cmd, output=e.output, stderr=e.stderr)
         except IOError as e:
             raise IOError(e)
+
+    def get_workdir(self):
```

Comment:
As discussed in call, move to `verify_invoke.py` since it is only used for that script.